addon/styles/addon.scss

```diff
@@ -1,36 +1,59 @@
 @import 'node_modules/ember-frost-core/addon/styles/frost-theme';
-$info-bar-height: 80px;
+$info-bar-height: 70px;
 $info-bar-border-width: 1px;
 $info-bar-padding-left: 30px;
 $info-bar-padding-right: 0;
 $info-bar-padding-top-bottom: 20px;
-$info-bar-title-margin-bottom: 8px;
+$info-bar-title-separator: 5px;
 
 .frost-info-bar {
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
   align-items: center;
-  height: $info-bar-height + $info-bar-border-width;
-  box-sizing: border-box;
+  height: $info-bar-height;
   border-bottom: $info-bar-border-width solid $frost-color-lgrey-1;
   background-color: $frost-color-white;
   padding: $info-bar-padding-top-bottom $info-bar-padding-right $info-bar-padding-top-bottom $info-bar-padding-left;
 
-  .title {
-    line-height: 1;
+  .icon {
```

Comment:
We should update all of the `.icon` type classes to something like `.frost-info-bar-icon`